src/components/Table/Table.stories.tsx

```diff
@@ -173,6 +168,7 @@ export const WithControls: Story = {
       columnHeaders={args.useColumnStyles ? columnHeadersAlt : columnHeaders}
       columnStyles={args.useColumnStyles ? columnStylesComplex : undefined}
       tableData={args.useColumnStyles ? tableDataAlt : tableData}
+      fontSize="8px"
```

Comment:
Were you just testing here with `8px`?

Reply:
Oops. Yes. A silly use case, but doable.